core/package/__tests__/core-package.test.js

```diff
@@ -273,6 +274,37 @@ describe("Package", () => {
       );
     });
   });
+
+  describe(".updateLocalDependency()", () => {
+    it("works with workspace: protocols", () => {
```

Comment:
I didn't have any direct tests of this method here because it is repeatedly tested in the publish and version tests. This is good to have, though.